### PR TITLE
9 projects page

### PR DESCRIPTION
**Pull request by 4dambarr**
Setup projects page

I suggest merging this one, then sorting out the conflicts between gatsby and the blog pages, so the projects page can be deployed asap

### CODE REVIEW

File: src/components/ProjectCard.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  font-style: italic;
+`;
+
+const ProjectPreview = styled(Img)`
+  border: 5px solid #555;
+`;
+
+import Button from "./Button";
+
+const ProjectCard = ({ project }) => {
```
Comment (benslv) [inline src/components/ProjectCard.js:26]: Can you destructure the props out a little bit here so there's not loads of `project.frontmatter...` used throughout the rest of it? Would clean up the code a little bit.

File: src/components/ProjectCard.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  const displayPreview = () => {
```
Comment (benslv) [inline src/components/ProjectCard.js:27]: I think there's probably a better way to do the conditional rendering on this preview image. Are you basically aiming to check whether a preview image exists, and rendering a message if not?

Could use a ternary operator which looks at `preview.childImageSharp.fixed` (which will be truthy if it exists but falsey if not)?

File: src/templates/project.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+const Title = styled.h1``;
+const Client = styled.h2``;
+const Lead = styled.h3``;
+const Developers = styled.h3`
+  font-weight: normal;
+`;
+
+const Template = ({ data }) => {
+  const { markdownRemark } = data; // data.markdownRemark holds your post data
+  const { frontmatter, html } = markdownRemark;
```
Comment (benslv) [inline src/templates/project.js:17]: Can you destructure out the frontmatter props here? Like above, will make the rest of the code cleaner to read.

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (benslv) [inline .vscode/settings.json:1]: Probably should add this to the .gitignore, as it's related more to your own IDE instead of the project.

**Review by benslv — CHANGES_REQUESTED** [review]

File: src/components/ProjectCard.js (hunk 2 above)
Comment (4dambarr) [reply src/components/ProjectCard.js:27]: I had a go at that but it wasn't working, may have related to the issues I was having with the query though so will try a ternary operator again

File: src/components/ProjectCard.js (hunk 1 above)
Comment (4dambarr) [reply src/components/ProjectCard.js:26]: What do you actually mean by destructuring? Do you have an example?

File: src/components/ProjectCard.js (hunk 2 above)
Comment (4dambarr) [reply src/components/ProjectCard.js:27]: Resolved this by checking whether project.frontmatter.preview exists, as opposed to the image itself.

**Review by benslv — APPROVED** [review]

**Pull request merged**